.github/workflows/build-push.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.8
+
+      - name: Build
+        run: go build -v ./...
```

Comment:
We want to run the test as well, right?

Reply:
Yes, thats true! Will this run the docker specific test ? It seems to only run `run: go test -v ./...`

Reply:
It will run the test, because the test is just alongside the other tests in the test folder. But the test is written to exit gracefully if the image isn't available - see the discussion with Tudor below.